docs/docs/core/admin/link-strapi-design-system.md

```diff
@@ -0,0 +1,25 @@
+# Linking the Strapi Design System
+
+Follow these steps to use a local version of the Strapi design system with the Strapi monorepo
+
+First, run `yarn build` in `strapi-design-system/packages/strapi-design-system` to generate the bundle.
+
+In your copy of Strapi, you can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).
+
+### Relative path
+
+Replace the version number in both `strapi/packages/core/admin/package.json` and `strapi/packages/core/helper-plugin/package.json` with the relative path to your copy of the design system:
+
+```
+"@strapi/design-system": "link:../../../../strapi-design-system/packages/strapi-design-system"
+```
```

Comment:
Could you add paths for both cases? 🙈 I think this would make it much easier for devs because they can just copy it.

Reply:
You mean relative paths from the docs to the files in core?

Reply:
The relative link: path, yeah.

Reply:
oh right gotcha

Reply:
Actually the relative link: path should be the same for both admin and helper-plugin assuming the design system is at the same level as the strapi monorepo